core/domain/event_services_test.py

```diff
@@ -290,13 +515,23 @@ def setUp(self):
             self.save_new_valid_exploration('exp_id', self.admin_id))
 
     def test_average_ratings_of_users_exps_are_calculated_correctly(self):
+
+        admin_average_ratings = (
+            user_services.get_dashboard_stats(self.admin_id)['average_ratings'])
+        self.assertEqual(admin_average_ratings, None)
```

Comment:
Should probably be assertIsNone here and elsewhere.

Reply:
(Probably not worth changing though, IMO. Let's just get this merged once the tests pass!)

Reply:
I updated this and updated the branch with develop branch as well :-)